src/main/resources/db/migration/V1_0__create_initial_referral.sql

```diff
@@ -0,0 +1 @@
+create table referral (id uuid not null, created timestamp, primary key (id));
```

Comment:
Since we are going to store draft referrals in here, does it make sense to call it `draft_referrals`?

Reply:
i left a comment on another PR with some more discussion on this point. we can dicuss in that thread.

https://github.com/ministryofjustice/hmpps-interventions-service/pull/23#discussion_r537589834

Reply:
👍 commented there, thanks!